.github/workflows/release.yml

```diff
@@ -25,4 +26,4 @@ jobs:
           sed -i 's/__PACKAGE_VERSION__/${{ github.event.release.tag_name }}/g' zanshincli/version.py setup.py
           rm CLI.md
           make CLI.md
-          #make pypi
+#         make pypi
```

Comment:
Can we remove this, then?

Reply:
No, this was temporary as I was testing running the release workflow during CI, and didn't want to push to PyPI while I was iterating. It will be removed before I ask for a review. :)